src/ChangeLeadTimeGraph.tsx

```diff
@@ -35,7 +35,7 @@ interface ProcessRepository {
 }
 
 export const composeGraphData = (_: ChartProps, data: DoraRecord[]) => {
-  let reduced = data.reduce(
+  return data.reduce(
```

Comment:
I'm in general not a fan of doing any sort of logic as part of a return statement.  It only saves 1 line of code but when trying to debug can cause headaches trying to step through things and set breakpoints.